src/runners/scilla_runner.ml

```diff
@@ -115,6 +115,36 @@ let rec output_event_json elist =
     | _ -> `Null :: j)
   | [] -> []
 
+let validate_get_init_json init_file gas_remaining sourcever =
```

Comment:
Could you change `sourcever` into `source_ver` to make the naming scheme more consistent?

Reply:
Consistent with what?

Reply:
with the two other parameters both using an underscore: `init_file` and `gas_remaining`

Reply:
I parsed at first it as `sourc`_`ever` :)